doc/i2p.md

```diff
@@ -47,10 +56,13 @@ information in the debug log about your I2P configuration and connections. Run
 `bitcoin-cli help logging` for more information.
 
 It is possible to restrict outgoing connections in the usual way with
-`onlynet=i2p`. I2P support was added to Bitcoin Core in version 22.0 (mid-2021)
-and there may be fewer I2P peers than Tor or IP ones. Therefore, using
-`onlynet=i2p` alone (without other `onlynet=`) may make a node more susceptible
-to [Sybil attacks](https://en.bitcoin.it/wiki/Weaknesses#Sybil_attack). Use
+`onlynet=i2p` (combined with `-onion=0` if disabling outbound onion connections
+is desired; see [doc/tor.md](tor.md)).
+
+I2P support was added to Bitcoin Core in version 22.0 (mid-2021) and there may
```

Comment:
Mid-2021? 22.0 is not released yet.

Reply:
22.0 is the first release that `doc/i2p.md` will be in.

Reply:
If the release takes place in August or early September, "mid-2021" seems ok to me but can change or remove if controverserial.  It wasn't added by this PR.

Reply:
Yes, I would not call September mid-2021, more like "autumn 2021", but OTOH I also not use where "mid-year" begins or ends (June or July would be definitely mid-2021 I think). Original schedule for 22.0 final release was August 1, but we are already past that.

Reply:
Removed `(mid-2021)`